src/prefect/utilities/storage.py

```diff
@@ -82,3 +83,35 @@ def extract_flow_from_file(
                 return exec_vals[var]
 
     raise ValueError("No flow found in file.")
+
+def extract_flow_from_module(
+    module_str: str, flow_name: str = None
+) -> "Flow":
+    """
+    Extract a flow object from a python module.
+
+    Args:
+        - module_str (str, optional): A module path pointing to a .py file containing a flow.
```

Comment:
```suggestion
        - module_str (str): A module path pointing to a .py file containing a flow.
```

Reply:
Another good catch! I'll go back through the docs on the Local storage class as well and update everything there.